src/main/java/com/google/cloud/spanner/pgadapter/statements/IntermediatePortalStatement.java

```diff
@@ -94,9 +93,9 @@ public DescribeMetadata describe() {
       ResultSet statementResult = connection.executeQuery(this.statement);
       setStatementResult(0, statementResult);
       return new DescribePortalMetadata(statementResult);
-    } catch (SpannerException e) {
-      logger.log(Level.SEVERE, e, e::getMessage);
```

Comment:
Removed logging, as an invalid user query is not something that we should be logging (it's not a system fault).